pages/about/about.md

```diff
@@ -3,22 +3,36 @@ title: About
 ---
 
 ## Who is the ELIXIR SPLASH for?
-The ELIXIR Training SPLASH has been designed to guide life scientists in their efforts to <>. It is based on the various steps of the training lifecycle, although not all the steps will be relevant to everyone.
+The ELIXIR Training SPLASH has been designed to guide trainer in their efforts to plan, design, develop, deliver and evaluate training activties. It is based on the various steps of the training lifecycle, although not all the steps will be relevant to everyone.
 
-The contents are generated and maintained by the ELIXIR Training platform [ELIXIR-TrP](https://elixir-europe.org/). More information on how we want to sustain and govern ELIXIR Training SPLASH can be found on <>.
+The contents are generated and maintained by the ELIXIR Training platform [ELIXIR-TrP](https://elixir-europe.org/). We want to sustain and govern ELIXIR Training SPLASH via the ELIXIR Programme 2024+.
+
+PSLASH is short for 
+
+* Skills,
+* Professional development,
+* Learning Assessment,
+* Support and
+* Help
+
+along the training life cycle.
 
 ## What does the ELIXIR Training SPLASH aim to achieve?
 
-<>
+The project aims to establish a digital environment representing a “one-stop-shop” for ELIXIR Training Platform (ETrP) services in order to increase its visibility, accessibility and usage and enable more participation within and outside the ELIXIR community.
+
+The ELIXIR Training Platform has several successful training products, such as TeSS, the Train-the-Trainer programme, the FAIR training project, relevant papers, a training operational handbook containing guidelines etc. We aim to transform access to these resources by creating a single entry point for trainers, trainees, training providers, funders, training developers, and the other ELIXIR Platforms and Communities.
+
+SPLASH will adopt a user-centric approach to present the information sought in a ready-to-use format that will enable the packaging and visibility of the different training products, using the successful model of the RDMkit, and the ELIXIR Toolkit theme. The created digital resource will be connected with the ELIXIR website and TeSS with the possibility to connect to other services in the future.
 
 ## Funding acknowledgement
-ELIXIR Training SPLASH was developed with funding from BioHackathon2023 and the ELIXIR 2024-2028 programme. We are grateful to all who contributed their time to producing contents for ELIXIR Training SPLASH. 
+ELIXIR Training SPLASH was developed with funding from ELIXIR BioHackathon Europe 2023 and the ELIXIR programme 2024-2028. We are grateful to all who contributed their time to producing contents for ELIXIR Training SPLASH. 
 
 ## How to cite ELIXIR Training SPLASH 
 
 <div class="card bg-light my-4">
   <div class="card-body">
-    <p class="card-text"><i>ELIXIR (2023)</i></p>
+    <p class="card-text"><i>ELIXIR SPLASH Pushing the boundaries of training development (2023)</i></p>
```

Comment:
The year here is static, which means it needs manual updating. Could this be replaced with something dynamic that will always look up-to-date?
```suggestion
    <p class="card-text"><i>ELIXIR SPLASH Pushing the boundaries of training development (2023)</i></p>
```

Reply:
I'll open another issue to address this request.